emojirades/commands/gamestate_commands/inferred_correct_guess_command.py

```diff
@@ -80,10 +83,32 @@ def execute(self):
             )
 
         yield (None, f"<@{state['winner']}>++")
+
+        # Build the score message
+        if score == 1000:
+            prefix = (
+                f"Ok <@{state['winner']}> just give up already, you've won the game"
+            )
+        elif score == 500:
+            prefix = f":tada::tada: Ladies and gentlemen he's daym done it again :tada::tada:"
+        elif score == 400:
+            prefix = f":trophy: This is a big milestone <@{state['winner']}>, you should feel proud"
+        elif score == 300:
+            prefix = f"Third century's the charm they say <@{state['winner']}>, congrats :sunglasses:"
+        elif score == 200:
+            prefix = f"Not going to lie <@{state['winner']}> this is pretty impressive"
+        elif score == 100:
+            prefix = f"Triple digits <@{state['winner']}>! Not everyone makes it this far! :tada:"
+        elif score % 50 == 0:
+            prefix = f"Another day, another 50 point milestone for <@{state['winner']}> :chart_with_upwards_trend:"
+        else:
+            prefix = f"Congrats <@{state['winner']}>"
```

Comment:
come on! this is duplicated! extract into a function or something? and add unit tests! :P

Reply:
or maybe this whole inferred class could extend the other one.

Reply:
Yeah..... was kinda cbf on that.... raise #204 to track this